components/external-services-mock/internal/auditlog/configurationchange/config.go

```diff
@@ -7,10 +7,10 @@ import (
 )
 
 func InitConfigurationChangeHandler(router *mux.Router, handler *ConfigChangeHandler) {
-	router.HandleFunc("/search", handler.SearchByString).Methods(http.MethodGet)
+	router.HandleFunc("/search", handler.SearchByTimestamp).Methods(http.MethodGet)
 	router.HandleFunc("", handler.Save).Methods(http.MethodPost)
 	router.HandleFunc("", handler.List).Methods(http.MethodGet)
 	router.HandleFunc("/{id}", handler.Get).Methods(http.MethodGet)
 	router.HandleFunc("/{id}", handler.Delete).Methods(http.MethodDelete)
-	router.Path("/search").Queries("query", "*").HandlerFunc(handler.SearchByString).Methods("GET")
+	router.Path("/search").Queries("timeFrom", "*", "timeTo", "*").HandlerFunc(handler.SearchByTimestamp).Methods("GET")
```

Comment:
I wonder why should we have this combination of path and query params additionally specified? Won't those kind of requests get caught up by the handler on line 10. The HandlerFunc is looking for queryparams which are snake cased ("time_from" and "time_to") so those would not even be processed.

Reply:
I removed this as it was unnecessary